Apps/GatewayApi/src/Controllers/PhsaController.cs

```diff
@@ -59,9 +58,9 @@ public PhsaController(IDependentService dependentService)
         [HttpGet]
         [Authorize(Policy = UserProfilePolicy.Read)]
         [Route("dependents/{hdid}")]
-        public RequestResult<IEnumerable<DependentModel>> GetAll(string hdid)
+        public ActionResult<IEnumerable<DependentModel>> GetAll(string hdid)
         {
-            return this.dependentService.GetDependents(hdid);
+            return this.Ok(this.dependentService.GetDependents(hdid));
         }
```

Comment:
I had done [something similar](https://github.com/bcgov/healthgateway/pull/4599/commits/a071f02d1e6a72a3a258c42a5ff4133322f34221) for the AgentAccessController but reverted it once I learned it was not needed and it made it more difficult to determine the response type, both manually and in Swagger.  (I think you still have a RequestResult involved here but it is not complaining that the types don't match.)

Reply:
I respectfully disagree :) IMO this is a better way to control response code, types and swagger documentation. @MikeLyttle let's discuss, I'd be interested to know why you think implicit return types are better.

Reply:
I linked the wrong part of my sentence.  The linked commit I referenced was the revert and it was initiated [at your request](https://github.com/bcgov/healthgateway/pull/4599#discussion_r1063804055).

Reply:
It doesn't check the type when returning ActionResult.